src/RewardsStreamerMP.sol

```diff
@@ -188,13 +187,6 @@ contract RewardsStreamerMP is
 
         VaultData storage vault = vaultData[msg.sender];
 
-        if (lockPeriod > 0) {
-            if (vault.totalLockTime + lockPeriod > MAX_LOCKUP_PERIOD) {
-                revert StakingManager__InvalidLockPeriod();
```

Comment:
I don't think this error is used anywhere anymore, so it can be removed from the contract.